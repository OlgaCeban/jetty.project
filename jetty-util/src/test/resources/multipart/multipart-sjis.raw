--VA2isIGmYNkgC3qrrGXnlQcWO17WB3a4npVQey
Content-Disposition: form-data; name="japanese"
Content-Type: text/plain; charset=Shift_JIS
Content-Transfer-Encoding: 8bit

Œ’Ž¡
--VA2isIGmYNkgC3qrrGXnlQcWO17WB3a4npVQey
Content-Disposition: form-data; name="hello"
Content-Type: text/plain; charset=Shift_JIS
Content-Transfer-Encoding: 8bit

ƒƒƒ†?ƒ^
--VA2isIGmYNkgC3qrrGXnlQcWO17WB3a4npVQey--
