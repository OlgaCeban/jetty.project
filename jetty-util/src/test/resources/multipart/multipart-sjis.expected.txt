Content-Type|multipart/form-data; boundary="VA2isIGmYNkgC3qrrGXnlQcWO17WB3a4npVQey"
Parts-Count|2
Part-ContainsContents|japanese|健治
Part-ContainsContents|hello|ャユ戆タ
